lerna-package-lock.json

```diff
@@ -2049,20 +2063,20 @@
 			}
 		},
 		"@fluid-experimental/property-dds": {
-			"version": "2.0.0-internal.2.1.2",
-			"resolved": "https://registry.npmjs.org/@fluid-experimental/property-dds/-/property-dds-2.0.0-internal.2.1.2.tgz",
-			"integrity": "sha512-PaNtohuioQ/jpYcFwVu5t302AdeYmlaU2kfhEDoLGTqmRx3GNW1NemyV8cB2isXlZODeJGVOnu697pQf3Rq++w==",
+			"version": "2.0.0-internal.2.2.1",
```

Comment:
Pretty sad that our back compat tests are testing against the future instead of the past. Makes them confusing and not very useful. I wonder if we should just disable them.